portality/datasets.py

```diff
@@ -1,570 +1,48 @@
-# -*- coding: UTF-8 -*-
-# the comment above is for the Python interpreter, there are Unicode
-# characters written straight into this source file
-
-import json
+import pycountry
 import sys
-import os
 
 if sys.version_info[0] == 2 and sys.version_info[1] < 7:
     from portality.ordereddict import OrderedDict
 else:
     from collections import OrderedDict
 
-# countries
-with open(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", 'country-codes.json'), 'rb') as f:
-    countries = json.loads(f.read())
-countries_dict = OrderedDict(sorted(countries.items(), key=lambda x: x[1]['name']))
-countries = countries_dict.items()
-
-country_options = [('','')]
+# Gather the countries with 2-character codes
+country_options = [('', '')]
 country_options_two_char_code_index = []
 
+for co in sorted(pycountry.countries, key=lambda x: x.name):
```

Comment:
And actually, lets gather these code blocks into functions at least, and then call them, so it's a bit cleaner.  Like:

```
country_options = gather_country_options()
```